src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -2699,6 +2705,11 @@ private bool HasImplicitFunctionTypeToFunctionTypeConversion(FunctionTypeSymbol
             var sourceDelegate = sourceType.GetInternalDelegateType();
```

Comment:
>GetInternalDelegateType [](http://example.com/codeflow?start=44&length=23)

Were all other call sites of this API reviewed and properly adjusted to handle nulls?

Reply:
Yes, I have reviewed all use of `GetInternalDelegateType()`.